### ISSUE TITLE
Docs build fails with an ESMValTool conflict

### ISSUE THREAD

**Issue by Peter9192** [eWaterCycle/ewatercycle#30]
I [updated the RTD](https://github.com/eWaterCycle/ewatercycle/pull/22), but the build fails as you can see [here](https://readthedocs.org/projects/ewatercycle/builds/13411348/). It seems there's a conflict which might be caused by ESMValTool?

```
The conflict is caused by:
    esmvaltool 2.2.0 depends on matplotlib>3.3.1
    esmvaltool 2.1.1 depends on ESMPy
    esmvaltool 2.1.0 depends on ESMPy
    esmvaltool 2.0.0 depends on ESMPy
```

Could it be due to the python version in the [readthedocs.yml](https://github.com/eWaterCycle/ewatercycle/blob/master/readthedocs.yml) which is out of sync? @sverhoeven @nielsdrost @SarahAlidoost ?

**Comment by Peter9192** [comment]
Tried updating to python3.8 [here](https://github.com/eWaterCycle/ewatercycle/pull/31) but the next problem is cartopy.

**Comment by nielsdrost** [comment]
In the GitHub action that runs the test, we now use conda for some of the dependencies: https://github.com/eWaterCycle/ewatercycle/pull/23

I guess we also need that for the readthedocs setup, or mock them. For an example of mocking see the ESMValTool readthedocs config: https://github.com/ESMValGroup/ESMValTool/blob/master/doc/sphinx/source/conf.py

**Comment by SarahAlidoost** [comment]
> I [updated the RTD](https://github.com/eWaterCycle/ewatercycle/pull/22), but the build fails as you can see [here](https://readthedocs.org/projects/ewatercycle/builds/13411348/). It seems there's a conflict which might be caused by ESMValTool?
> 
> ```
> The conflict is caused by:
>     esmvaltool 2.2.0 depends on matplotlib>3.3.1
>     esmvaltool 2.1.1 depends on ESMPy
>     esmvaltool 2.1.0 depends on ESMPy
>     esmvaltool 2.0.0 depends on ESMPy
> ```
> 
> Could it be due to the python version in the [readthedocs.yml](https://github.com/eWaterCycle/ewatercycle/blob/master/readthedocs.yml) which is out of sync? @sverhoeven @nielsdrost @SarahAlidoost ?

I got similar errors while working on GitHub actions. The dependencies of esmvaltool need to be installed, as @nielsdrost mentioned in [this comment](https://github.com/eWaterCycle/ewatercycle/issues/30#issuecomment-812642804). For the github actions, we chose the installation of the dependencies via a conda environment as the environment.yml file is already available in the repo, see #23.

**Comment by Peter9192** [comment]
Thanks both. I removed the package installation for now and mocked all imports.
